rt_rubocop_defaults.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubocop", "~> 1.2"
+  spec.add_dependency "rubocop", "~> 1.4"
```

Comment:
TODO: Specify the version which fixes https://github.com/rubocop-hq/rubocop/issues/9095 here.